util/chunk/mutrow.go

```diff
@@ -346,3 +346,27 @@ func setMutRowJSON(col *column, j json.BinaryJSON) {
 	copy(col.data[1:], j.Value)
 	col.offsets[1] = int32(dataLen)
 }
+
+// ShadowCopyPartialRow use shadow copy to instead of AppendPartialRow,
+// ShadowCopyPartialRow copies the data of row to the first row of dst.
```

Comment:
Oh, dst means destination.  Well, dst also means daylight saving time. 
We can only use the abbreviation when there is a clear context.